lib/main_common.pm

```diff
@@ -458,7 +458,9 @@ sub load_extra_tests() {
         # dependency of git test
         loadtest "console/sshd";
         loadtest "console/zypper_ref";
-        loadtest "console/zypper_info";
+        if (sle_version_at_least('12-SP2') || check_env('DISTRI', 'opensuse')) {
```

Comment:
Hello @SergioAtSUSE  this changes are becasue https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2678 , main part is enabling tests for released products. 

second, I'll tomorrow change this PR to run on SLE12SP1 with first part of this test, and remove change from main.pm as @sysrich adviced me